pango/Setup.hs

```diff
@@ -14,13 +15,18 @@ import Distribution.Verbosity
 import Distribution.Simple.Utils
 import System.FilePath
 
+#if MIN_VERSION_Cabal(2,0,0)
+import Distribution.Version ( versionNumbers )
+import Distribution.Types.PackageName ( unPackageName )
+#endif
+
 main =
   defaultMainWithHooks gtk2hsUserHooks {
 
     postConf = \args cf pd lbi -> do
       let verb = (fromFlag (configVerbosity cf))
       cPkgs <- getPkgConfigPackages verb lbi pd
-      let [pangoVersion] = [ v | PackageIdentifier (PackageName "pango") v <- cPkgs ]
+      let [pangoVersion] = [ v | PackageIdentifier (unPackageName -> "pango") v <- cPkgs ]
```

Comment:
https://hackage.haskell.org/package/Cabal-1.24.0.0/docs/Distribution-Package.html

It was a field, but has been converted to a function in Cabal 2.0.0.2